macro/run_simScript.py

```diff
@@ -557,13 +556,14 @@
 # -----Start run----------------------------------------------------
 run.Run(options.nEvents)
 # -----Runtime database---------------------------------------------
-kParameterMerged = ROOT.kTRUE
-parOut = ROOT.FairParRootFileIo(kParameterMerged)
-parOut.open(parFile)
-rtdb.setOutput(parOut)
-rtdb.saveOutput()
-rtdb.printParamContexts()
-getattr(rtdb,"print")()
+if not options.DarkPhoton:
```

Comment:
So far, I saw this error in the pbrem case. However, I thought it would be better to apply it to all DP productions.
How should we proceed with this problem?

Reply:
@ThomasRuf what do you think? As a workaround it's fine, but I think we should try to understand why the `rtdb` causes a crash/error. Do we actually use it for any purpose currently?

Reply:
i am lost. what is the pbrem case error?

Reply:
Hi Thomas,
You investigated this error last year(27 Feb, 2019) and title was " Error in Dark Photon Production (in Proton Bremsstrahlung Process)";


```FairShip/macro/run_simScript.py --DarkPhoton -A pbrem --mass 0.88 --epsilon 1.5848932e-07 -n 2```

...

```run.Run(2)```
...
>>>Debug: decay product of A: 2 -2
>>>[INFO] ship event 0 / pythia event-nr 0
>>>Debug: decay product of A: 13 -13

```kParameterMerged = ROOT.kTRUE```
```parOut = ROOT.FairParRootFileIo(kParameterMerged) >>> parOut.open(parFile)```

>>>True

```rtdb.setOutput(parOut)```

>>>True

``` rtdb.saveOutput()```

>>>*** Break *** segmentation violation

Reply:
I agree with Oliver, would be better to understand what is the reason for the problem, rather than to hide it.